specification/redisenterprise/resource-manager/Microsoft.Cache/preview/2024-09-01-preview/redisenterprise.json

```diff
@@ -1817,22 +2257,88 @@
       },
       "description": "The response of a list-all operation."
     },
+    "AccessPolicyAssignmentProperties": {
+      "title": "Redis Enterprise database access policy assignment properties",
+      "description": "Properties of Redis Enterprise database access policy assignment.",
+      "type": "object",
+      "properties": {
+        "provisioningState": {
+          "$ref": "#/definitions/ProvisioningState",
+          "readOnly": true,
+          "description": "Current provisioning status of the access policy assignment."
+        },
+        "accessPolicyName": {
+          "type": "string",
+          "description": "Name of access policy under specific access policy assignment. Only default policy is supported.",
+          "default": "default",
+          "pattern": "^([a-zA-Z0-9][a-zA-Z0-9-]*[a-zA-Z0-9]|[a-zA-Z0-9])$"
+        },
+        "user": {
+          "type": "object",
+          "properties": {
+            "objectId": {
+              "type": "string",
+              "description": "The object ID of the user.",
+              "example": "6497c918-11ad-41e7-1b0f-7c518a87d0b0"
+            }
+          },
+          "description": "The user associated with the access policy."
+        }
+      },
+      "required": [
+        "user"
+      ]
+    },
+    "AccessPolicyAssignment": {
+      "description": "Describes the access policy assignment of Redis Enterprise database",
+      "type": "object",
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/AccessPolicyAssignmentProperties",
+          "description": "Properties of the access policy assignment."
+        }
+      },
+      "allOf": [
```

Comment:
what does this "allOf" property do?

Reply:
https://swagger.io/docs/specification/data-models/oneof-anyof-allof-not/

Reply:
(inheritance!)